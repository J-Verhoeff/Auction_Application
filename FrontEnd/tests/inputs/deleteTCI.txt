login
admin
delete
admin
delete
userNA
delete
user
logout
exit