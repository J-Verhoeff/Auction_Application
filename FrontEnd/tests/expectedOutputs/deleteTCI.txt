
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Username: Cannot delete current user!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Username: User not found!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Username: User successfully deleted!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: Goodbye!
